src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -46,6 +46,8 @@ public class GitTestTemplate {
     protected static final String TEST_COMMIT_HASH_PARENT = "c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151";
     protected static final String MAIN_AUTHOR_NAME = "harryggg";
     protected static final String FAKE_AUTHOR_NAME = "fakeAuthor";
+    protected static final String UNCONVENTIONAL_AUTHOR_NAME = "-invalidGitUsername_TreatedAsUnknownUser";
```

Comment:
I personally still suggest changing this name since it does not reflect the new logic. You will need to submit a pull request to modify https://github.com/reposense/testrepo-Alpha/blob/master/annotationTest.java to address that

Reply:
I have changed the names to better reflect the new authors and their respective names. As a result there should be issues with the automated checks on GitHub. I have made a [separate PR](https://github.com/reposense/testrepo-Alpha/pull/14) to the testrepo. Thanks for your suggestion!

Reply:
The 2 failing tests are found in `AnnotatorAnalyzerTest.java`, as the names now do not match with what is found in `annotationTest.java`.